### DISCUSSION TITLE
Allow administration access without Development access

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6417] [Ideas]
![Screenshot 2022-06-22 at 10 28 09 AM](https://user-images.githubusercontent.com/1195573/174930298-7766f113-88a8-4942-a40c-859d092a3e62.jpg)

We may need a project manager/secretary/admin/non-technical people to handle emails invites and users onboarding etc. But we don't want them to have access to edit the apps which is super dangerous. 

I tried to turn off Development Access and turn on Administration Access like the screenshot but that has no effect. 

We may have to rebuild the whole user onboarding journey just to achieve the effect that admin has no permissions to editing apps.

Kind of hope that this will be a feature in the roadmap?

**Comment by melohagan** [comment]
Good idea! There is a larger epic on the roadmap to revamp the RBAC, so this could be part of that.
